### DISCUSSION TITLE
因為爸爸的一句話！牙醫師跨入 AI 領域打造「牙醫版美圖秀秀」，入選 500 Global

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#666] [科技新知]
成立於2020年的Preteeth運用AI演算法，只要牙醫拍下病人臉部，就能馬上分析病人的微笑，並模擬牙齒美白、陶瓷貼片等療程後的結果，讓病人預先清楚了解治療後的模樣，降低與牙醫之間的溝通成本。
只要開啟Preteeth手機應用程式或網頁，載入病人的照片，Preteeth就會自動偵測牙齒範圍，讓牙醫選取需要的功能，為病人客製化模擬出治療的前後差異。

https://technews.tw/2022/06/11/preteeth-ai/

**Comment by yucheng1028** [comment]
蠻有趣的，之前做牙齒都不知道做完會長怎樣

**Comment by SamTU0222** [comment]
曾經有在新聞上看到有人使用掃描口腔來對建模，這樣反而更加方便了

**Comment by treeann** [comment]
AI真的太多用途

**Comment by stan890425** [comment]
這樣在手術前就可以看看符不符合自己預期了~~

**Comment by sys511613** [comment]
感覺可以應用到各種手術上，甚至是剪頭髮，這樣就可以在做之前先看到之後會變怎麼樣，再做適合自己的決定

**Comment by jcpan821** [comment]
若是能運用在其他手術，像是骨折治療就太好了

**Comment by alexthebigpp** [comment]
感覺很棒，可以讓人事先看到結果

**Comment by yenwei12138** [comment]
發展成熟的話，人人都能有一口漂亮牙齒了，剩下就只有預算問題(也有可能原本就只有這問題?

**Comment by bloodtrail0511** [comment]
太酷了吧

**Comment by etin2468** [comment]
讓患者能先看到樣貌，我覺得蠻棒的

**Comment by Jimmy911218** [comment]
讓人可以提前知道做完後的模樣感覺不會這成麻煩了
